lib/qubes-rpc-multiplexer

```diff
@@ -51,14 +51,7 @@ for DIR in $QREXEC_SERVICE_PATH; do
 done
 IFS=$ifs
 
-if [ -x "$CFG_FILE" ] ; then
-    # shellcheck disable=SC2086
-	exec "$CFG_FILE" ${QREXEC_SERVICE_ARGUMENT}
-	echo "$0: failed to execute handler for" "$1" >&2
-	exit 1
-else
-    # shellcheck disable=SC2086
-	exec /bin/sh -- "$CFG_FILE" ${QREXEC_SERVICE_ARGUMENT}
-	echo "$0: failed to execute handler for" "$1" >&2
-	exit 1
-fi
+# shellcheck disable=SC2086
+exec "$CFG_FILE" ${QREXEC_SERVICE_ARGUMENT}
+echo "$0: failed to execute handler for $1" >&2
+exit 126
```

Comment:
126 is used by qrexec-client-vm for "request refused". Please try to avoid conflicting with it. You can simply preserve the old value (1).